translate/misc/test_optrecurse.py

```diff
@@ -16,3 +17,14 @@ def test_splitext(self):
         root = os.path.join(dirname, name)
         print(fullpath)
         assert self.parser.splitext(fullpath) == (root, extension)
+
+    def test_outputfile_receives_bytes(self, capsys):
+        parser = optrecurse.RecursiveOptionParser({"txt": ("po", None)})
+
+        temp_file = TemporaryFile()
+        with TemporaryFile() as tempfile:
+            out = parser.openoutputfile(None, temp_file.name)
+            out.write(b'binary suff')
+
+        out = parser.openoutputfile(None, None)  # To sys.stdout
```

Comment:
@claudep does this mean that it is necessary to change how to run tests on Travis?